webhook.rst

```diff
@@ -0,0 +1,284 @@
+Webhook
+=======
+
+.. versionadded:: 6.3
+
+    The Webhook component was introduced in Symfony 6.3 and is marked
+    as experimental.
+
+The Webhook component aims at easing the setup of callback functions
+through HTTP in your application.
+
+Webhooks allow third-party services to communicate with your application
+without you having to make a request to the third-party application.
+By providing a webhook URL to the remote service, it will be able to
+send a request to your URL when a predefined event occurs in their system.
+Unlike an API, it is the third-party service that defines the payload that
+will be sent to your own endpoint. It is therefore up to you to adapt to
+the service to handle such payload accordingly.
+
+As an example of use case, some mail providers set up a service allowing
+you to retrieve the status of one or more emails you have sent
+and thus know if they have been delivered or not.
+These services then use your webhook to send you this
+information once they have it, so that you can store this information
+and process it on your side.
+
+Installation
+------------
+
+You can install the Webhook component with:
+
+.. code-block:: terminal
+
+    $ composer require symfony/webhook
+
+.. include:: /components/require_autoload.rst.inc
+
+Basic Usage
```

Comment:
I'd add a small 'Configuration' section  after this with a short description, and a small example:
`.. configuration-block::

    .. code-block:: yaml

        # config/packages/framework.yaml
        framework:
            webhook:
                routing:
                    email:
                        service: App\Webhook\MailerWebhookParser # custom parser id
                        secret: '%env(EXTERNAL_SERVICE_SECRET)%'` # this can be validated in the doPrase method

Reply:
I don't know if we should add this in the component documentation or in the framework documentation? 🤔

Reply:
From what I've seen on other documentation pages, this is included in the component doc, not in the framework doc

Reply:
Added :+1: